signac/core/synced_collections/synced_list.py

```diff
@@ -185,8 +185,8 @@ def reset(self, data=None):
 
     def __setitem__(self, key, value):
         self._validate(value)
-        with self._load_and_save():
-            with self._suspend_sync():
+        with self._load_and_save:
+            with self._suspend_sync:
```

Comment:
I believe these can be written more concisely as:
```suggestion
        with self._load_and_save, self._suspend_sync:
```

Reply:
Supported by Python > 3.1: https://docs.python.org/3/reference/compound_stmts.html#the-with-statement
> With more than one item, the context managers are processed as if multiple with statements were nested:
>
> ```python
> with A() as a, B() as b:
>     SUITE
> ```
> is semantically equivalent to:
>
> ```python
> with A() as a:
>     with B() as b:
>         SUITE
> ```

Reply:
Good catch, I'm aware of this construct but have never actually seen it in production use anywhere so I always forget that it's an option.